src/material-examples/cdk/stepper/cdk-custom-stepper-without-form/cdk-custom-stepper-without-form-example.ts

```diff
@@ -20,4 +20,7 @@ export class CustomStepper extends CdkStepper {
   onClick(index: number): void {
     this.selectedIndex = index;
   }
+
+  static ngAcceptInputType_linear: boolean | string;
+  static ngAcceptInputType_selectedIndex: number | string;
```

Comment:
Can we add a comment for these two members then? Otherwise it might be confusing in the docs?

Reply:
Done.